1_For_Loops.ipynb

```diff
@@ -202,7 +202,7 @@
   },
```

Comment:
<p>solve this without the passed variable and use the one liner syntax of if statements, for example:</p><p><code>def return_true_if_1(x):</code></p><p>	<code>return True if x==1 else False</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RB-Data-Analysis-Python-Course/pfc-week04-beckyrosenfeld/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>